doc/api/fs.md

```diff
@@ -3162,6 +3162,22 @@ system requests but rather the internal buffering `fs.readFile` performs.
    the call to `fs.readFile()` with the same file descriptor, would give
    `'World'`, rather than `'Hello World'`.
 
+### Performance Considerations
+
+The `fs.readFile()` method asynchronously reads the contents of a file into
+memory one chunk at a time, allowing the event loop to turn between each chunk.
+This allows the read operation to have less impact on other activity that may
+be using the underlying libuv thread pool but means that it will take longer
+to read a complete file into memory.
+
+For applications that require as-fast-as-possible reading of file contents, it
+is better to use `fs.read()` directly and for application code to manage
+reading the full contents of the file itself.
+
```

Comment:
Is it possible to `quantify` the `slowness` in some way, so as to give a rough guidance on the API selection? for example, `a chunk size can be up to 512 KB, so file sizes smaller than that will be read in one iteration`?

Reply:
Added some comments on the chunk size.